pymongo/event_listeners.py

```diff
@@ -0,0 +1,130 @@
+import logging
```

Comment:
This is missing the copyright notice. Please copy it from any other source file and update the copyright date.